src/main/java/objects/Mob.java

```diff
@@ -1,12 +1,25 @@
 package objects;
 
-import java.util.ArrayList;
-
-//TODO complete this class
 public class Mob {
-    public ArrayList<String> getDrops() {
-        //TODO
-        return null;
+    String name;
+    int combatLvl;
+    Item[] drops;
```

Comment:
Required in String format of the name, as the mob class will have no access to the Object of the item, just the name on drop tables